app/models/blog.rb

```diff
@@ -0,0 +1,6 @@
+class Blog < ApplicationRecord
```

Comment:
Look at adding some [validations](http://guides.rubyonrails.org/active_record_validations.html) on the fields for this and other classes. At the moment, I can create users without a name, blogs without a title, etc.

There may be instances where you want to include this sort of behaviour (e.g. allowing anonymous/guest blogging); however, I'd argue that this constitutes an exceptional use-case, and should be noted in the comments if this is the desired behaviour.

Reply:
Will do, thanks for the link.